src/app/app-routing.module.ts

```diff
@@ -4,13 +4,15 @@ import { PostsComponent } from "./posts/posts.component";
 import { PostPageComponent } from "./posts/post-page/post-page.component"
 import { LoginComponent } from "./auth/login/login.component";
 import { CreatePageComponent } from "./posts/create-page/create-page.component";
+import { NotFoundComponent } from "./shared/not-found/not-found.component";
 
 const routes: Routes = [
     {path: '', redirectTo: '/', pathMatch: 'full'},
     {path: '', component: PostsComponent},
     {path: 'post/:id', component: PostPageComponent },
     {path: 'login', component: LoginComponent},
-    {path: 'create', component: CreatePageComponent}
+    {path: 'create', component: CreatePageComponent},
+    { path: '**', component: NotFoundComponent}
 ];
```

Comment:
use linter for project/or add a formatter on file save

Reply:
Added prettier config in this PR: 
[PR10- Enhance app](https://github.com/hannapereverzieva/blog/pull/10)